Description: don't depend on obsolete syslog target (syslog is socket activated)
Author: Marc Haber <mh+debian-packages@zugschlus.de>
Forwarded: no
Last-Update: 2016-08-07
--- a/atopacct.service
+++ b/atopacct.service
@@ -2,7 +2,6 @@
 Description=Atop process accounting daemon
 Documentation=man:atopacctd(8)
 Conflicts=psacct.service
-After=syslog.target
 Before=atop.service
 
 [Service]
